website/common/script/libs/getItemInfo.js

```diff
@@ -52,7 +67,9 @@ export default function getItemInfo (user, type, item, officialPinnedItems, lang
 
   let itemInfo;
 
-  switch (type) { // eslint-disable-line default-case
+  switch ( // eslint-disable-line default-case
+    type
+  ) {
```

Comment:
this is odd, did the linter ask you to do this?

Reply:
It did! I couldn't get it to not throw an error unless I made the change. I suspect my linter may be using slightly stricter rules.